eng/scripts/Cadl-Project-Generate.ps1

```diff
@@ -42,20 +42,27 @@ $npmWorkingDir = Resolve-Path $tempFolder/$innerFolder
 $mainCadlFile = If (Test-Path "$npmWorkingDir/client.cadl") { Resolve-Path "$npmWorkingDir/client.cadl" } Else { Resolve-Path "$npmWorkingDir/main.cadl"}
 $resolvedProjectDirectory = Resolve-Path "$ProjectDirectory/src"
 
+$HasEmitterOutputDir = $false;
 if (Test-Path "$npmWorkingDir/cadl-project.yaml") {
     $cadlProjectYaml = Get-Content -Path "$npmWorkingDir/cadl-project.yaml" -Raw
     $yml = ConvertFrom-YAML $cadlProjectYaml
-    if ( $yml["emitters"]["@azure-tools/cadl-csharp"]["sdk-folder"]) {
-        $yml["emitters"]["@azure-tools/cadl-csharp"]["sdk-folder"] = ""
+    if ($yml -And $yml["options"] -And $yml["options"]["@azure-tools/cadl-csharp"] -And $yml["options"]["@azure-tools/cadl-csharp"]["emitter-output-dir"]) {
+        $HasEmitterOutputDir = $true
```

Comment:
`$yml["options"]["@azure-tools/cadl-csharp"]["emitter-output-dir"]` is not used afterwards. Shall we change to save it like below?
```suggestion
        $EmitterOutputDir = $yml["options"]["@azure-tools/cadl-csharp"]["emitter-output-dir"]
```

Reply:
We don't need the emitterOutpuDir value, I just need to support two scenaros, one is define emitteroutputdir in cadl-project.yaml, the other is not.